src/script/components/report-card.ts

```diff
@@ -40,25 +59,164 @@ export class ReportCard extends LitElement {
         background: white;
         box-shadow: 0 1px 4px 0px rgb(0 0 0 / 25%);
         border-radius: 50%;
-        color: var(--primary-color);
+        color: #5231A7;
+
+        width: 32px;
+      }
+
+      .flipper-button ion-icon {
+        pointer-events: none;
       }
 
       .flipper-button::part(control) {
         font-size: 22px;
       }
 
+      .flipper-button::part(content) {
+        padding: 0;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+      }
+
       .score-block {
         display: flex;
         align-items: center;
         justify-content: space-between;
       }
+
+      .options-button {
+        width: 217px;
+        float: right;
+        margin-right: 4em;
+      }
+
+      .options-button::part(underlying-button) {
+        background: white;
+        color: var(--font-color);
+      }
+
+      #overall-score #total-score {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+
+        margin-right: 1.8em;
+      }
+
+      #overall-score #total-score h4 {
+        font-size: var(--medium-font-size);
+      }
+
+      #package-block {
+        display: flex;
+        justify-content: flex-end;
+      }
+
+      #package-block fast-anchor {
+        width: 152px;
+        color: white;
+        box-shadow: var(--button-shadow);
+        border-radius: var(--button-radius);
+      }
     `;
   }
 
   constructor() {
     super();
   }
 
+  opened(targetEl: EventTarget) {
+    console.log(targetEl);
+
+    if (targetEl) {
+      const flipperButton = (targetEl as Element).classList.contains('flipper-button') ? (targetEl as Element) : (targetEl as Element).querySelector(
+        '.flipper-button'
+      );
+
+      if (flipperButton) {
+        if (flipperButton.classList.contains('opened')) {
+          
+          flipperButton.animate(
+            [
+              {
+                transform: 'rotate(0deg)'
+              }
+            ],
+            {
+              duration: 200,
+              fill: "forwards"
+            }
+          )
+
+          flipperButton.classList.remove('opened');
+        } else {
+          flipperButton.classList.add('opened');
+
+          flipperButton.animate(
+            [
+              {
+                transform: 'rotate(0deg)',
+              },
+              {
+                transform: 'rotate(90deg)',
+              },
+            ],
+            {
+              duration: 200,
+              fill: 'forwards',
+            }
+          );
+        }
+      }
+    }
+  }
+
+  handleManiScore(ev: CustomEvent) {
+    if (ev && ev.detail.score) {
+      this.maniScore = ev.detail.score;
+    }
+    else {
+      this.maniScore = 0;
+    }
```

Comment:
This can be shortened to:

```typescript
this.maniScore = ev?.detail?.score || 0;
```

Reply:
always forget about this syntax, so used to if / else statements for this kinda stuff haha. Resolved in a commit below!